packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx

```diff
@@ -209,18 +209,34 @@ function Footer({
           isDisabled={editingField}
           onPress={onClickRemainingSplit}
         >
+          {}
           <SvgSplit width={17} height={17} />
           <Text
             style={{
               ...styles.text,
               marginLeft: 6,
             }}
           >
-            Amount left:{' '}
-            {integerToCurrency(
-              transaction.amount > 0
-                ? transaction.error.difference
-                : -transaction.error.difference,
+            {!emptySplitTransaction ? (
+              <>
+                <Trans>Add New Split</Trans>
```

Comment:
This should be sentence case and i think this should use a formatted translation string e.g. `t('Add new split - {{amount}} left', { amount: integerToCurrency(...) })`